tests/forms/contact.py

```diff
@@ -35,27 +38,35 @@ def test_contact_form(admin, rf, mailoutbox):
     assert message.from_email == settings.DEFAULT_FROM_EMAIL
     reply_to = u'%s <%s>' % (data['name'], data['email'])
     assert reply_to == message.extra_headers['Reply-To']
-    subject = u'[%s] %s' % (settings.POOTLE_TITLE, data['email_subject'])
+    assert [recipient_email] == message.recipients()
+    assert message.subject.startswith(u'[%s] ' % settings.POOTLE_TITLE)
```

Comment:
Not strictly necessary, for this line, but it is an error that haunted us for some time so I wanted it to be explicit. What you suggest is next line, btw.